Firestore/core/src/firebase/firestore/model/field_value.h

```diff
@@ -69,7 +69,7 @@ class FieldValue {
     // position instead, see the doc comment above.
   };
 
-  FieldValue() : tag_(Type::Null) {
+  FieldValue() {
```

Comment:
Optional: this now can be `= default`.

Reply:
Doesn't seem to work... (Due to the union with nontrivial types.)

eg, this fails to compile:
```
#include <string>

class Abc {
 public:
  Abc() = default;

  union {
    std::string c;
  };
};
    
int main() {
  Abc abc;
}
```